_posts/2012-04-18-app.markdown

```diff
@@ -35,11 +36,26 @@ permalink: app
 
 </div>
 
+Ao longo deste guia, você verá pedaços de código formatados da seguinte forma:
+
+{% highlight sh %}
+exemplo de código
+{% endhighlight %}
+
+<div>
+
+
+<p>Isso significa que o texto destacado é um código e provavelmente precisa ser executado ou inserido em um dos seus arquivos de projeto. O ícone ao lado do texto destacado permitirá que você saiba qual ferramenta usar.</p>
+<p>Por exemplo, se você ver um ícone de terminal ao lado do código formatado, como no exemplo acima, será necessário copiar o código e colá-lo no Terminal (Mac) / Prompt de Comando (Windows).</p>
+</div>
+
+Precisa de alguns lembretes pelo caminho? Leia [Cola útil para Ruby, Rails, console etc.](http://www.pragtob.info/rails-beginner-cheatsheet/) (em inglês)
+
 ### Importante
 
 É importante que você conheça a sintaxe de comandos específica para seu Sistema Operacional - os comandos a executar em computadores rodando Windows podem apresentar pequenas diferenças sintáticas em relação àqueles usados em ambientes Mac ou Linux.  Se você não estiver seguro com a sintaxe dos comandos use o link para o sistema operacional, que está localizado  após os comandos mostrados neste tutorial. Se você estiver usando um serviço na nuvem  (por exemplo: Codenvy), você deverá executar os comandos conforme a sintaxe Linux ainda que esteja em ambiente Windows.

```

Comment:
Seria interessante colocar uma observação que o tutorial foca na sintaxe de linux/mac, para que os desavisados com windows, já de início, busquem o link com os comandos adequados.

Reply:
Essa já tá na imagem :)